src/linux/system.rs

```diff
@@ -763,7 +761,21 @@ fn get_exe_name(p: &Process) -> String {
         .unwrap_or_else(String::new)
 }
 
-fn get_system_info(info: &str) -> Option<String> {
+enum InfoType {
+    /// The end-user friendly name of:
+    /// - Android: The device model
+    /// - Linux: The distibutions name.
+    Name,
+    OsVersion,
+}
+
+#[cfg(target_os = "linux")]
```

Comment:
Nitpicking but I would write the condition as follow: `#[cfg(not(target_os = "android"))]`

Reply:
Switched 👍